### ISSUE TITLE
Training hyper-parameters for DCAE

### ISSUE THREAD

**Issue by francois-rozet** [mit-han-lab/efficientvit#146]
Hi :wave:

I am having trouble training a DCAE from scratch. Could you provide the training details (optimizer, learning rate, batch size, weight decay, number of epochs, ...) for each phase (1,2,3)?

Thank you very much for your work!

**Comment by alexblattner** [comment]
have you managed yet?
